crates/ignore/src/default_types.rs

```diff
@@ -173,7 +173,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("red", &["*.r", "*.red", "*.reds"]),
     ("robot", &["*.robot"]),
     ("rst", &["*.rst"]),
-    ("ruby", &["Gemfile", "*.gemspec", ".irbrc", "Rakefile", "*.rb"]),
+    ("ruby", &["config.ru", "Gemfile", "*.gemspec", ".irbrc", "Rakefile", "*.rb", "*.rbw"]),
```

Comment:
Thanks! Could you wrap this to 79 columns (inclusive) like other long lines in this file?

Reply:
@BurntSushi of course :)

Thanks for being so reactive!